recipes/paramiko/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{%set name = "paramiko" %}
+{%set version = "1.17.2" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "d436971492bf11fb9807c08f41d4115a82bd592a844971737a6a8e8900c4677c" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pycrypto >=2.1,!=2.4,<3.0
+    - ecdsa >=0.11,<2.0
+
+  run:
+    - python
+    - pycrypto >=2.1,!=2.4,<3.0
+    - ecdsa >=0.11,<2.0
+
+test:
+  imports:
+    - paramiko
+
+about:
+  home: http://www.paramiko.org/
+  home: https://github.com/paramiko/paramiko/
+  license: LGPL 2.1
+  license_file: LICENSE
+  summary: 'SSH2 protocol library'
+  license_family: Public-Domain
+  dev_url: https://github.com/paramiko/paramiko/
+  doc_url: http://docs.paramiko.org/
+
+extra:
+  recipe-maintainers:
+    - pmlandwehr
```

Comment:
Please add @vamega based on our [discussion]( https://github.com/conda-forge/staged-recipes/pull/1156/files#r75807235 ) and myself.